extensions.json

```
@@ -105,6 +105,11 @@
       "version": "0.3.2",
       "checkout": "0.3.2"
     },
+    {
+      "id": "castwide.solargraph",
+      "repository": "https://github.com/castwide/vscode-solargraph",
+      "version": "0.22.0"
```

It seems ok to publish `castwide.solargraph` from the default branch without pinning a specific version:

```suggestion
      "repository": "https://github.com/castwide/vscode-solargraph"
```